packages/plugin-cloudflare/src/tunnel.ts

```diff
@@ -122,7 +126,7 @@ class TunnelClientInstance implements TunnelClient {
         }
         // If already resolved, means that the CLI already received the tunnel URL.
         // Can't retry because the CLI is running with an invalid URL
-        if (resolved) throw processCrashed()
+        if (resolved) throw processCrashed(error.message)
 
         outputDebug('Cloudflared tunnel crashed, restarting...')
```

Comment:
Not only for the `resolved` case, I think we should output `error.message` in the `outputDebug` as well, for the cases where the URL is not resolved yet (most of them)